lib/web_components_rails/html_import_processor.rb

```diff
@@ -6,7 +6,50 @@
 #   https://github.com/rails/sprockets/blob/3.x/lib/sprockets/directive_processor.rb
 class WebComponentsRails::HTMLImportProcessor
 
-  VERSION = '9'
+  VERSION = '10'
+  URI_ATTRIBUTES = %w(
+    action
+    cite
+    data
+    formaction
+    href
+    itemid
+    manifest
+    ping
+    poster
+    src
+  ).freeze
+  BOOLEAN_ATTRIBUTES = %w(
```

Comment:
Can you put a comment about where you got these lists from? (maybe a link to some w3c site?)